docs/wallet_browser_extension/definition.md

```diff
@@ -0,0 +1,34 @@
+---
+tags:
+  - wallet
+  - browser extension
+  - provider API
+---
+
+# Definition
+
+## Dapp
+
+A decentralized application (dapp) is an application built on a decentralized network that combines a smart contract and a frontend user interface.
+
+## Provider
+
+A JavaScript object made available to a consumer, that provides access to wallet by means of a Client.
```

Comment:
should be "_to a user_"?

Reply:
The content is referenced from https://eips.ethereum.org/EIPS/eip-1193.
The client is short for client side.